### PR TITLE
Load definitions

### PR DESCRIPTION
**Pull request by daurer**
depends on #7, #8 and #9

### CODE REVIEW

File: charts/rabbitmq/templates/rabbitmq-load-definition-sealed-secret.yaml [hunk 1]
```diff
@@ -0,0 +1,8 @@
+apiVersion: bitnami.com/v1alpha1
+kind: SealedSecret
+metadata:
+  name: {{ .Release.Name }}-load-definition
+  namespace: {{ .Release.NameSpace }}
```
Comment (garryod) [inline charts/rabbitmq/templates/rabbitmq-load-definition-sealed-secret.yaml:5]:
```suggestion
  namespace: {{ .Release.Namespace }}
```

**Review by garryod — CHANGES_REQUESTED** [review]

File: charts/rabbitmq/templates/rabbitmq-load-definition-secret.yaml [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+metadata:
+  name: {{ .Release.Name }}-load-definition
+  namespace: {{ .Release.Namespace }}
+type: Opaque
+stringData:
+  load_definition.json: |-
+    {
+      "users": [
+        {
+          "name": "user",
```
Comment (garryod) [inline charts/rabbitmq/templates/rabbitmq-load-definition-secret.yaml:12]: Should we call the admin `admin`?

File: rabbitmq-app.yaml [hunk 3]
```diff
@@ -12,8 +12,29 @@ spec:
...
+      name: rabbitmq-certs
+      namespace: messagebus
+      jqPathExpressions:
+        - '.data."ca.crt"'
+        - '.data."tls.crt"'
+        - '.data."tls.key"'
+    - kind: Secret
+      name: rabbitmq
+      namespace: messagebus
+      jqPathExpressions
```
Comment (garryod) [inline rabbitmq-app.yaml:32]:
```suggestion
      jqPathExpressions:
```

**Review by garryod — CHANGES_REQUESTED** [review]

File: charts/rabbitmq/templates/rabbitmq-erlang-secret.yaml [hunk 4]
```diff
@@ -0,0 +1,8 @@
+apiVersion: bitnami.com/v1alpha1
+kind: SealedSecret
+metadata:
+  name: {{ .Release.Name }}-erlang-secret
+  namespace: {{ .Release.Namespace }}
+spec:
+  encryptedData:
+    rabbitmq-erlang-cookie: AgAJ7RciGKjdfZlneoc9eD79hD3aCPVCRrUjhw7az7+hDaOLpXQpCJK+N7rnmCvJUIcqPbA7yQTURF6gGW5tcPyVb1bt2iEzImpgIRoFWTm7/pEmJqEuxPWuLZhZP48fPj+39rRX4LHJikDStGDSQdrsOTehQQ0Av5CfTBLDVDJ+DpUeFdsilLniwkGBhTklLrjeCRe7HROG+sSJsqKZGEuFV7KTt1HH60PSWZbkv59ax4W8d6sxhem900XW91QPRNXwl7929V+yXKmD3aZbQo+vIirdxeoaYhvHgjCDCS6kOhBa2VJo7byXEXFHy2FDtkRZyHba0xdoWm5fjzUILFwW9rU3VKZcoxfreonfQ9SC5H8Cbg30FAfjz6xClbqZw3iFCkl9LYDzpYn3ajAztyZ+5XnnkOnd7tXnnjuyWCXomqZvvrGtRI+kDannWA96PX6KlhqtRJSyignuT9ElIx05USxklNl+RVPBDP2+Jl11suoKBcSQs3P+azs3YZVWGoeBPslX2VjF9cIdtaMqO4MF3TF+Ec3H7PkZOddk3BfD1t7pFSERkpwqnLd2moteTvBDEmzhEtaNxBq+RFyyRK695s/HetOa3q3kLEjdBNGSk+Qj0iQ7B+OEypgXwNg8JAB5JsYLrjH9qn4RHDzW0N4Ri0fh3vHlQ1+Mg2mwOI+VMbUuxmXky2/kQZnwIOh3rm4x1G5IKi7Pp7rhONmeShAAgsgwZWkIPsAJvLxIjIkR9mHsxqDELpN0
```
[the remark this answers is not in the document]
Comment (daurer) [reply charts/rabbitmq/templates/rabbitmq-erlang-secret.yaml:8]: Yes, that would make sense. Maybe we could make a separate PR for this. see #13

File: charts/rabbitmq/templates/rabbitmq-load-definition-secret.yaml (hunk 2 above)
Comment (daurer) [reply charts/rabbitmq/templates/rabbitmq-load-definition-secret.yaml]: done

File: charts/rabbitmq/templates/rabbitmq-load-definition-secret.yaml [hunk 5]
```diff
@@ -0,0 +1,53 @@
+apiVersion: v1
+kind: Secret
+metadata:
+  name: {{ .Release.Name }}-load-definition
+  namespace: {{ .Release.Namespace }}
+type: Opaque
+stringData:
+  load_definition.json: |-
```
[the remark this answers is not in the document]
Comment (daurer) [reply charts/rabbitmq/templates/rabbitmq-load-definition-secret.yaml]: done

**Pull request merged**